e2e/tests/functional/plugins/notebook/notebookWithCouchDB.e2e.spec.js

```diff
@@ -221,6 +221,41 @@ test.describe('Notebook Network Request Inspection @couchdb', () => {
 
         expect(filterNonFetchRequests(addingNotebookElementsRequests).length).toBeLessThanOrEqual(4);
     });
+
+    test('Search tests', async ({ page }) => {
```

Comment:
Let's add an annotation referencing the issue. We can even link to `openmct-yamcs` , no problems there

Reply:
```js
        test.info().annotations.push({
            type: 'issue',
            description: 'https://github.com/akhenry/openmct-yamcs/issues/69'
        });
```